.kitchen.yml

```diff
@@ -60,6 +64,31 @@ suites:
           checkpoint: true
           switch_user: yes
 
+- name: attributes
```

Comment:
Oh, I see what you mean! I had not realized there was already so much similar config.

Since they're pretty much the same, it would be great if they all aliased the same anchor. That would make this config a lot smaller.